app/javascript/App/lib/redux/createFilterReducer.js

```diff
@@ -9,8 +9,12 @@ const createFilterReducer = ({ namespace, filterList }) => (
   if (namespace !== action.namespace) return state
   const { type, payload } = action
   const filterIndex = (key) => listIndex(state.get('activeFilters'), key)
-  const filterObject = (key, value) =>
-    filterList.find((o) => o.get('key') === key).set('value', fromJS(value))
+
+  const filterObject = (key, value) => {
+    if (value === 'undefined') return
+    var fromJSCopy = fromJS
```

Comment:
Shall we stick with `let` or `const` moving forward and avoid `var`s since we're in ES6 now?